locales/en/kafkaoverview.json

```diff
@@ -0,0 +1,14 @@
+{
+  "contactSalesCardCallToActionButton": "Contact sales",
+  "contactSalesCardMainText": "Contact sales to get a pre-paid subscription or modify limits of your current subscription. Pre-paid subscriptions are available to customers outside the United States and Canada.",
```

Comment:
Please don't hate me, but I pasted the wrong text previously. This is the final text:
> Contact sales to get a pre-paid subscription or modify limits of your current subscription. Pre-paid subscriptions are available to customers in any location, including locations outside the United States and Canada.